src/carplay.cs

```diff
@@ -806,11 +822,8 @@ interface CPWindow {
 		[Export ("mapButtonSafeAreaLayoutGuide")]
 		UILayoutGuide MapButtonSafeAreaLayoutGuide { get; }
 
-#if false
-// Apple is trying to use CPTemplateApplicationScene but it is not exposed. https://github.com/xamarin/maccore/issues/1822
```

Comment:
don't forget to update/close that issue (it can be done at the same time as the PR, no need to wait for it to merge since it won't change Apple's headers)